racingcar-java/src/main/java/racingcar5/Cars.java

```diff
@@ -0,0 +1,38 @@
+package racingcar5;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Cars {
+
+    private List<Car> cars;
+
+    public Cars(MoveStrategy moveStrategy, String names) {
+        List<String> carNames = NameSpliter.nameSplit(names);
+        cars = IntStream.range(0, carNames.size())
+                .mapToObj(i -> new Car(moveStrategy, carNames.get(i)))
+                .collect(Collectors.toList());
+    }
+
+    public void moveCars() {
+        IntStream.range(0, cars.size())
+                .forEach(i -> cars.get(i).move());
+    }
+
+    public Car getCar(int index) {
+        validateIndex(index);
+        return cars.get(index);
+    }
+
+    public int getSize() {
+        return cars.size();
+    }
+
+    private boolean validateIndex(int index) {
+        if(index < 0 || index >= cars.size()) {
+            throw new IllegalArgumentException("잘못된 인덱스 접근");
+        }
+        return true;
```

Comment:
```suggestion
    private void validateIndex(int index) {
        if(index < 0 || index >= cars.size()) {
            throw new IllegalArgumentException("잘못된 인덱스 접근");
        }
```
 boolean을 반환할 필요는 지금당장은 없을 것 같네요

Reply:
확인했습니다!
`void`로 바꾸었습니다 👍